orange/src/main/java/com/hartwig/hmftools/orange/algo/cuppa/CuppaDataFactory.java

```diff
@@ -100,4 +103,13 @@ static int getSvFeatureValue(@NotNull CuppaPredictions cuppaPredictions, @NotNul
 
         return (int) Math.round(predictionEntry.FeatureValue);
     }
+    static CuppaMode getCuppaMode(@NotNull CuppaPredictions cuppaPredictions) {
+        if (cuppaPredictions.MainCombinedClassifierName == ClassifierName.COMBINED) {
+            return CuppaMode.WGTS;
+        } else if (cuppaPredictions.MainCombinedClassifierName == ClassifierName.DNA_COMBINED) {
+            return CuppaMode.WGS;
+        } else {
+            return CuppaMode.UNKNOWN;
```

Comment:
Since we now we are saying that this should be impossible, maybe we should throw an exception here (IllegalArgumentException) if the input is not COMBINED/DNA_COMBINED?